lib/money/version.rb

```diff
@@ -1,3 +1,3 @@
 class Money
-  VERSION = '6.15.0'
+  VERSION = '6.15.1'
```

Comment:
this is new functionality, not a bug fix, so you will want to bump the minor version to `6.16.0`